backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/dao/selection/QuerySelectionHelper.java

```diff
@@ -211,15 +211,14 @@ public <T, ET extends T> TypedSqlQuery<ET> createUUIDSelectionQuery(
 		final String selectionWhereClause = "s.ZZ_UUID=?";
 		final String selectionOrderBy = "s." + SELECTION_LINE_ALIAS;
 
-		final TypedSqlQuery<ET> querySelection = new TypedSqlQuery<>(
+		return new TypedSqlQuery<>(
 				ctx.getCtx(),
 				clazz,
 				selectionWhereClause,
 				ctx.getTrxName())
 						.setParameters(querySelectionUUID)
 						.setSqlFrom(selectionSqlFrom)
-						.setOrderBy(selectionOrderBy);
-
-		return querySelection;
+						.setOrderBy(selectionOrderBy)
+				.setOrderBy(SELECTION_LINE_ALIAS);
```

Comment:
@metas-ts  why calling setOrderBy twice?

Reply:
hadn't seen it and thought i was missing 🤦‍♂️ 
thx!